documentation/getting-started.md

```diff
@@ -2,29 +2,40 @@
 layout: docs-cat
 description: 'Getting started with WoTT'
 title: 'Getting Started with the WoTT Agent and Dashboard'
-full_description: We'll take you from startup to installation of the WoTT Agent and registering for the Dashboard so you can begin integrating security for your needs.
+full_description: We'll take you from startup to installation of the WoTT Agent so you can begin integrating security for your needs.
 title_content: 'Getting Started'
 redirect_from:
   - /getting-started-1
   - /getting-started
   - /blog/categories/getting-started
 ---
 
-WoTT's primary goal is to simplify and manage end-to-end security for developers across a multitude of internet platforms. We manage the credentials and authentication for your Edge/IoT devices and servers- WoTT secures your projects so that you don't have to.
+WoTT's primary goal is to help developers manage and resolve security risks for linux instances by identifying vulnerabilities and providing clear instructions to resolve them. We also manage the credentials and authentication for your servers. WoTT secures your projects so that you don't have to. WoTT works on any linux system including edge devices.
 
-How do we do this? We offer you:
+How do we do this?
 
-* Simplified encryption of point-to-point communication
-* Cryptographic identity (x509 certificate issued from our CA)
-* Enabling the removal of hard coded credentials from web applications and firmware
+1. Recommended actions and developer guides to fix vulnerabilities
+2. Insights on vulnerabilities and metrics to manage the risk of your linux instances
+3. Simplified encryption of point-to-point communication and cryptographic identity (x509 certificate issued from our CA)
+4. Enabling the removal of hard coded credentials from web applications and firmware
 
 To access all these features, you will need to install the [WoTT Agent and Dashboard](https://dash.wott.io) to manage your various nodes through a simple and user-friendly interface. 
 
 We'll provide a step-by-step walkthrough of set-up and installation of the WoTT agent. It's hassle-free and simple to implement.
 
-## Installing the WoTT agent and claiming your device
+## Why you should consider DevSecOps:
 
-First, register for a [WoTT Dashboard](https://dash.wott.io) account and login. We have several packages available for your convenience. See our [prices]({{site.url}}/pricing) for more information.
+1. Most data breaches are caused by poor security hygiene (known CVEs, poor configuration, weak authentication).
```

Comment:
Let's use `*` instead of lists as i'm not sure the formatting works well for lists.

Reply:
Ok tried it your way.